ir/v1.def

```diff
@@ -475,11 +475,13 @@ class AttributeRef : Expression {
 class CompilerOptions;
 #end
 
-class V1Program {
+class V1Program : ISimpleNamespace {
```

Comment:
This code does not seem to be invoked anywhere, so there are essentially no test cases.